src/skin.cpp

```diff
@@ -460,11 +445,10 @@ void SkinInfillAreaComputation::generateInfill(SliceLayerPart& part, const Polyg
         offset_from_inner_wall += extra_perimeter_offset - innermost_wall_line_width / 2;
     }
     Polygons infill = part.insets.back().offset(offset_from_inner_wall);
-
-    infill = infill.difference(skin.offset(infill_skin_overlap));
+    infill = infill.difference(skin);
```

Comment:
this will create extra copying

why not:

```c++
part.infill_area = part.insets.back().offset(offset_from_inner_wall).difference(skin);
part.infill_area.removeSmallAreas(MIN_AREA_SIZE);
```